wordpress-mobile/WordPress-iOS
# WordPressAuthenticator update: Added `What is WordPress.com` button in `Continue With WordPress.com` flow

WordPressAuthenticator PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/646

To test:
- Launch the app and tap on `Login or sign up with WordPress.com` button.
- Validate that there is no `What is WordPress.com` button displayed.
- Ensure that the UI is presented as before.
Please confidence check that login UI is presented as before!

PR submission checklist:
